paasta_tools/utils.py

```diff
@@ -971,8 +984,12 @@ def get_branch_dict(self, service, branch):
         return self.get(full_branch, {})
 
 
+def get_paasta_branch_from_identifier(identifier):
```

Comment:
I feel like we should just call it `deploy_group` instead of `identifier`, and thus `get_paasta_branch_from_deploy_group`.